rpcs3/Emu/System.cpp

```diff
@@ -1160,6 +1160,17 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 			fs::file card_2_file(vfs::get("/dev_hdd0/savedata/vmc/" + argv[2]), fs::write + fs::create);
 			card_2_file.trunc(128 * 1024);
 		}
+		else if (m_cat == "PE" && from_hdd0_game)
```

Comment:
All PE games are digital by the way. The physical versions just have packages for the digital versions.

Reply:
Perhaps this comment is best inlined in the code. There are far too many magic strings and numbers floating around as it is.

Reply:
It's kinda obvious though, and tbh the comment confuses more than it helps

Reply:
It is not obvious, and the comment does not confuse more than it helps. It is non-trivial that all those games are digital; it is quite reasonable to wonder if this were an assumed hack. Having that errata tied to the code saves time in the future.

Reply:
What should they else be?
It's physically impossible to put a umd into the ps3

Reply:
And theres literally a category for physical disc games.

Reply:
PSP Remasters are distributed physically on PS3 BDs, while PSP Minis are distributed digitally on PSN, that's why I made the comment